docs/data/xrootd/install-standalone.md

```diff
@@ -97,13 +97,64 @@ If you do not need any of the following special configurations, skip to
 
 #### Enabling Hadoop support (EL 7 Only)
 
-For documentation on how to export your Hadoop storage using XRootD please see
-[this documentation](/data/xrootd/install-storage-element#optional-adding-hadoop-support-to-xrootd)
+Hadoop File System (HDFS) based sites should utilize the `xrootd-hdfs` plugin to allow XRootD to access their storage:
+
+1. Install the XRootD HDFS plugin package:
+
+        :::console
+        root@host # yum install xrootd-hdfs
+
+1. Add the following configuration to `/etc/xrootd/xrootd-clustered.cfg`:
+
+        :::file
+        ofs.osslib /usr/lib64/libXrdHdfs.so
+
+For more information, see [the HDFS installation documents](/data/install-hadoop).
+
+
+#### Enabling multi-user support
+
+By default XRootd servers write files on the storage system ad the Unix user `xrootd`, not as the user [authenticated](data/xrootd/xrootd-authorization) as.
```

Comment:
```suggestion
By default XRootD servers write files on the storage system aa the Unix user `xrootd` instead of the [authenticated](xrootd-authorization) user.
```